config/katello.migrations/201117205851-add-import-export-params.rb

```diff
@@ -0,0 +1,4 @@
+if answers['foreman_proxy_content'].is_a?(Hash)
```

Comment:
I wonder if we need to account for a possible case where `answers['foreman_proxy_content'] == true` rather than being a hash. I think some other migration would still run first and convert it to a hash but I'm not 100% positive.

Reply:
All the other migrations I see use the same check, `if answers['foreman_proxy_content'].is_a?(Hash)`.  The only exception is config/katello.migrations/200123161606-enable-pulpcore.rb:

```rb
  unless answers['foreman_proxy_content']
    answers['foreman_proxy_content'] = {} unless answers['foreman_proxy_content'].is_a?(Hash)
    answers['foreman_proxy_content']['proxy_pulp_isos_to_pulpcore'] = false
  end
```

This block would only run if it were `false` or `nil`, and would leave it unchanged in the event it were `true`.

Using `answers['foreman_proxy_content'].is_a?(Hash)` as a check seems reasonable to me, since it's also used by so many other migrations and I assume those don't break anything.